src/liger_kernel/ops/rms_norm.py

```diff
@@ -303,7 +302,6 @@ def backward(ctx, dY):
             X,
             W,
             r,
-            ctx.eps,
```

Comment:
does that mean we don't have to store eps in the forward pass?

Reply:
yes